Lib/fontParts/objects/base/validators.py

```diff
@@ -115,10 +115,11 @@ def validateLayerName(value):
 # -----
 
 def validateGlyphName(value):
-    """
-    XXX implement
-    """
-    return value
+    if not isinstance(value, basestring):
+        raise FontPartsError("Glyph names must be unicode strings, not %s." % type(value).__name__)
```

Comment:
Agreed. No further checks than if it's a string — but to @anthrotype's point, should this enforce the input to be a unicode string or just a string (unicode or not)?